src/newCity.js

```diff
@@ -0,0 +1,9 @@
+import getWeather from './getData';
+
+function newCity() {
+  const city = document.getElementById('getCity').value;
+  document.getElementById('getCity').value = '';
+  getWeather(city);
+}
+
```

Comment:
Generally in a click event method, you have to stop the propagation. That is, you should have something starting like thins
```js
function newCity(e) {
  e.preventDefault();
  // .....
  // .....
}
```